server/src/model.ts

```diff
@@ -1203,10 +1306,34 @@ export async function getAssignmentDataFromCode(
 export async function searchPublicContent(query: string) {
   // TODO: how should we sort these?
 
-  let query_words = query.split(" ");
-  let content = await prisma.content.findMany({
+  const query_words = query.split(" ");
+  const content = await prisma.content.findMany({
     where: {
-      AND: query_words.map((qw) => ({ name: { contains: "%" + qw + "%" } })),
+      AND: query_words.map((qw) => ({
+        OR: [
```

Comment:
I don't know what is more efficient in terms of the query, but you can make the description shorter by combining the code and name together:
```

  OR: [
    { name: { contains: "%" + qw + "%" } },
    {
      classifications: {
        some: {
          classification: {
            OR: [
              {
                code: { contains: "%" + qw + "%" },
                name: { contains: "%" + qw + "%" },
              },
            ],
          },
        },
      },
    },
  ],
```

Reply:
I think we should also search on category, and maybe even description. The code isn't much to go on. We'll eventually find a way to make all that information visible when browsing activities, as well.